Servers/UI/OJS.Servers.Ui/ClientApp/src/common/reduxNames.tsx

```diff
@@ -1,9 +1,17 @@
+// Submissions
+const submissionsSliceName = 'latestSubmissions';
```

Comment:
wouldn't it be a better approach if we have submissions slice, which in itself has latesSubmissions, since there could be few places where we fetch submissions data and so we can keep them in one property of the redux store, instead of having multiple submissions instances?

Reply:
This must be fixed on a global level, as currently the 'submissions' name clashes with the submissions slice in the administration

Reply:
you can change the administraions slice to administrationSubmissions and this will free the name submissions to be used in the client part of redux, without having to change anything more elsewhere